semantic-typescript/src/Language/TypeScript/Tags.hs

```diff
@@ -93,6 +93,19 @@ instance ToTags Ts.CallExpression where
         _ -> gtags t
       yield name = yieldTag name Call loc byteRange >> gtags t
 
+instance ToTags Ts.Module where
+  tags t@Ts.Module
+    { ann = loc@Loc { byteRange }
+    , name
+    } = match name
+    where
+      match expr = case expr of
+        Prj Ts.Identifier { text } -> yield text
+        -- TODO: Handle NestedIdentifiers and Strings
```

Comment:
Should we handle this TODO now or file an issue about it? (Also applies to the previous file.)